### PR TITLE
CP-2141-Topic Dialog Support

### PR DESCRIPTION
**Pull request by Azharhussain20**
**CP-2141-Topic Dialog Support**
_Highlight newly added topic to the channel config._
file changes: `CleverPush.m`, `CPTopicDialogCell.h`, `CPTopicDialogCell.m`, `CPTopicDialogCell.xib` ,`CPTopicsViewController.h`, `CPTopicsViewController.m`, `CPUtils.h` & `CPUtils.m`

### CODE REVIEW

File: CleverPush/CleverPush/Source/CPTopicsViewController.m [hunk 1]
```diff
@@ -260,9 +268,8 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
     
     UISwitch* deselectSwitch = [[UISwitch alloc] init];
     CGSize switchSize = [deselectSwitch sizeThatFits:CGSizeZero];
-    deselectSwitch.frame = CGRectMake(tableView.bounds.size.width - switchSize.width - 5.0f, (44 - switchSize.height) / 2.0f, switchSize.width, switchSize.height);
+    deselectSwitch.frame = CGRectMake(tableView.bounds.size.width - (switchSize.width + 5.0), (44 - switchSize.height) / 2.0f, switchSize.width, switchSize.height);
```
Comment (gbhrdt) [inline CleverPush/CleverPush/Source/CPTopicsViewController.m:271]: Can you please export the `44`, `5.0` and `2.0f` into constants / variables?

File: CleverPush/CleverPush/Source/CPTopicsViewController.m [hunk 2]
```diff
@@ -272,8 +279,10 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
     }
     
     UILabel* deselectEverything = [[UILabel alloc] init];
-    deselectEverything.text = [CPTranslate translate:@"deselectEverything"];
-    deselectEverything.frame = CGRectMake(10.0, (44 - switchSize.height) / 2.0f, tableView.bounds.size.width - (10 + switchSize.width), switchSize.height);
+    deselectEverything.text = @"Deselect Everything";
```
Comment (gbhrdt) [inline CleverPush/CleverPush/Source/CPTopicsViewController.m:282]: Why `[CPTranslate translate:@"deselectEverything"]` removed?

**Review by gbhrdt — CHANGES_REQUESTED** [review]

File: CleverPush/CleverPush/Source/CPTopicsViewController.m (hunk 2 above)
Comment (Azharhussain20) [reply CleverPush/CleverPush/Source/CPTopicsViewController.m:282]: I was testing with static text that's why I was there and forget to keep the original one.
I will replace with original one.

File: CleverPush/CleverPush/Source/CPTopicsViewController.m (hunk 1 above)
Comment (Azharhussain20) [reply CleverPush/CleverPush/Source/CPTopicsViewController.m:271]: Yes sure.

File: CleverPush/CleverPush/Source/CPTopicsViewController.m [hunk 3]
```diff
@@ -5,6 +5,11 @@
 #import "CPTranslate.h"
 #import "CPTopicDialogCell.h"
 
+static CGFloat const CPTopicHeight = 44;
+static CGFloat const CPTopicCellLeading = 5.0;
+static CGFloat const CPTopicHeightDevisor = 2.0f;
+static CGFloat const CPConstrains = 30.0;
```
Comment (gbhrdt) [inline CleverPush/CleverPush/Source/CPTopicsViewController.m:11]: Typo, please rename to `CPConstraints` with t

File: CleverPush/CleverPush/Source/CPTopicsViewController.m [hunk 4]
```diff
@@ -5,6 +5,11 @@
 #import "CPTranslate.h"
 #import "CPTopicDialogCell.h"
 
+static CGFloat const CPTopicHeight = 44;
+static CGFloat const CPTopicCellLeading = 5.0;
+static CGFloat const CPTopicHeightDevisor = 2.0f;
```
Comment (gbhrdt) [inline CleverPush/CleverPush/Source/CPTopicsViewController.m:10]: Devisor -> Divider

**Review by gbhrdt — CHANGES_REQUESTED** [review]

**Pull request merged**